research/conditional/utils/argparse.py

```diff
@@ -80,6 +80,20 @@ def introduce_parser_arguments(
     parser.add_argument("--train_dataset_path", type=str, default=None)
     parser.add_argument("--validation_dataset_path", type=str, default=None)
 
+    parser.add_argument("--final_eval_seed", type=int, default=1912)
+    parser.add_argument(
+        "--final_eval_dataloader_batch_size",
+        type=int,
+        default=64,
+        help="Batch size for final evaluation dataloader. It cannot be larger than the batch_size per GPU for the biggest model.",
+    )
+    parser.add_argument(
+        "--n_sequences_final_eval",
```

Comment:
I'm not sure if it's a good idea, but why don't we just specify `n_sequences_final_eval` and then choose `final_eval_dataloader_batch_size` dynamically as the biggest number that fits some constraints? We may be inefficient if we pick weird sequence number (like large prime naumber or something stupid), but it won't change the result.

Reply:
Discussed offline. See [this](https://github.com/llm-random/llm-random/pull/466#issuecomment-2457250987) comment.